x-pack/packages/kbn-slo-schema/src/schema/common.ts

```diff
@@ -59,6 +59,12 @@ const metaSchema = t.partial({
   }),
 });
 
+// TODO Kevin: add spaceId and change kibanaUrl to be optionally defined instead of backfilling it with an empty string?
```

Comment:
Should we make kibanaUrl optional instead of falling back into `''` when there is no kibanaUrl on the summary document (prior to 8.14)?